poc/run_poc.py

```
@@ -0,0 +1,218 @@
+        )
+
+    if verbose:
+        print(f"\n[Transformer ({out_t.latency_ms:.0f}ms)]")
+        print(f"  {out_t.output[:200]}{'...' if len(out_t.output) > 200 else ''}")
+        print(f"\n[SSM ({out_s.latency_ms:.0f}ms)]")
+        print(f"  {out_s.output[:200]}{'...' if len(out_s.output) > 200 else ''}")
+
+    result = compute_coherence_score(out_t.output, out_s.output)
+
+    if verbose:
+        print(f"\n[CoherenceScore]")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)